docs/dosya-sikistirma.md

```diff
@@ -6,8 +6,53 @@
 
 # Dosya Sıkıştırma
 
-> Eklenecek
+**Dosya sıkıştırma**, dijital dosyaların boyutunu azaltmak için kullanılan bir tekniktir. Bu işlem, bir veya birden fazla dosyanın kapladığı alanı daha küçük bir boyuta indirir. Dosya sıkıştırma, dosyaların daha az depolama alanı kaplamasını ve daha hızlı transfer edilmesini sağlar. Sıkıştırma, genel olarak verilerin tekrar eden veya gereksiz parçalarını kaldırarak veya veri yapısını optimize ederek yapılır. Bu işlem genellikle **ZIP**, **GZIP**, **RAR**, **7Z** gibi formatlar tercih edilerek gerçekleştirilir.
```

Comment:
> ve daha hızlı transfer edilmesini

Transfer hızına bir etkisi olacağını zannetmiyorum. Transfer edilecek dosya boyutu azalacağı için süre gene azalır fakat bu fark hızdan değil boyuttan kaynaklacaktır.

Reply:
Özellikle sunucu taraflı sistemlerde ve İçerik dağıtım ağlarında dosya sıkıştırma yöntemi, transfer süresinin kısalması ve veriminin artması amacıyla, transfer edilecek veri boyutunun azaltılması için tercih edilen bir yöntem olduğundan dolayı belirtmeye gerek duydum.

Reply:
sıkıştırma nedeniyle 100mbit olan internetiniz 200mbit olmaz. Süre azalır ama sıkıştırma nedeniyle değil. Dosya boyutu azaldığı için. İkisi arasındaki bağı iyi kuralım. Bu mantığa göre dosyanın yarısını silersem internetim 2x hızlanmış olur.

> Düzeltilmiş haliyle yeni bir Pull Request oluşturabilirim.

Yeni PR açmanıza gerek yok @DarkIsTaken forkladığınız repoya yeni commit gönderdiğinizde otomatik buraya da ekleniyor.